lib/arclight/custom_component.rb

```diff
@@ -46,9 +47,10 @@ def to_solr(solr_doc = {})
     def component_field_definitions
       [
         { name: 'level', value: formatted_level, index_as: :facetable },
-        { name: 'access_subjects', value: access_subjects, index_as: :facetable },
+        { name: 'access_subjects', value: access_subjects, index_as: :symbol },
         { name: 'containers', value: containers, index_as: :symbol },
-        { name: 'has_online_content', value: online_content?, index_as: :symbol }
+        { name: 'has_online_content', value: online_content?, index_as: :symbol },
+        { name: 'places', value: places, index_as: :facetable }
```

Comment:
Hi Darren, I assume you are talking about the line with 'places' which I've now removed. Thanks.